packages/ui/cypress/tests/watched-acccounts/watched-accounts.cy.ts

```diff
@@ -0,0 +1,62 @@
+import { addresses } from '../../fixtures/accounts'
+import { localHost } from '../../fixtures/landingData'
+import { landingPage } from '../../support/page-objects/landingPage'
+import { settingsPage } from '../../support/page-objects/settingsPage'
+
+describe('Watched Accounts', () => {
+  it('can add an account to the watch list', () => {
+    cy.visit(localHost)
+    landingPage.watchAccountButton().click()
+    settingsPage.accountAddressInput().type(`${addresses.Alice}{enter}`)
+    // autocomplete on the input seems to affect reliability so we wait a bit
+    cy.wait(1000)
+    settingsPage.accountNameInput().type('{selectall}{del}Alice')
+    settingsPage.addButton().click()
+    settingsPage.accountContainer().within(() => {
+      settingsPage.accountIcon().should('be.visible')
+      settingsPage.accountAddressLabel().should('be.visible')
+      settingsPage.accountNameLabel().should('be.visible')
+      settingsPage.accountDeleteButton().should('be.visible')
+    })
+  })
+
+  it('can remove an account from the watch list', () => {
+    cy.window().then((win) => {
+      win.localStorage.setItem('multix.watchedAccount', `["${addresses.AliceSS58}"]`)
```

Comment:
Same here, I think tests should be independant from the implementation details. In this case, adding an account is super quick and we should rather use the UI for this.

Reply:
Updated here too